lib/dependencies.go

```diff
@@ -15,3 +15,22 @@ type ProductionDependencies struct{}
 func (pd *ProductionDependencies) Disrupt(_ string) bool {
 	return false
 }
+
+/* Test dependencies */
```

Comment:
Is there a reason these are not just in `test/dependencies.go`? Theoretically we should only need the interface and ProductionDependencies in a neutral package and specific test dependencies can be created in the same files as the tests or a dedicated dependencies file in the test dir.

Reply:
If we put the dependencies under `test`, similar to what `skyd` is doing, those would be compiled in the binary.

If we put them in the test file where they are used (`api_test.go`) they won't be compiled in the binary but it would make them harder to reuse and it will generally make the project more disorganised.

I'll follow the `skyd` approach for now. Please let me know if you prefer the second approach.
I don't feel strongly about this, so I will do whatever you prefer.